packages/integration_test/ios/Classes/IntegrationTestIosTest.h

```diff
@@ -2,47 +2,40 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <Foundation/Foundation.h>
+@import Foundation;
+
+@class UIImage;
 
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol FLTIntegrationTestScreenshotDelegate;
+typedef void (^FLTIntegrationTestResults)(NSString *testName, BOOL success, NSString *_Nullable failureMessage);
 
 @interface IntegrationTestIosTest : NSObject
```

Comment:
Any chance we could fix the casing of "IOS" while we're here, or is that breaking to a public API?

Reply:
That's a good idea, and there's nothing iOS-y about it other than the `UIImage`.  I'll deprecate the entire `IntegrationTestIosTest` instead of just `testIntegrationTest:` and replace it with a `FLTIntegrationTestRunner` class.  It would be the same result if anyone was using this API (which the docs don't say to do, but it was technically possible).